homeassistant/components/volvooncall/__init__.py

```diff
@@ -314,6 +315,15 @@ def assumed_state(self):
         """Return true if unable to access real state of entity."""
         return True
 
+    @property
+    def device_info(self) -> DeviceInfo:
+        """Return a inique set of attributes for each vehicle."""
+        return DeviceInfo(
+            identifiers={(DOMAIN, self.vehicle.vin)},
+            name=self._vehicle_name,
+            model=self.vehicle.vehicle_type,
+        )
```

Comment:
```suggestion
        return DeviceInfo(
            identifiers={(DOMAIN, self.vehicle.vin)},
            name=self._vehicle_name,
            model=self.vehicle.vehicle_type,
            manufacturer="Volvo",
        )
```

Reply:
Yes, proposed this in a comment but were not sure it is allowed with constant return values. If so I'm positive to the change.

Reply:
You can add it

Reply:
Don't seem to have "write permissions" so had to do it in an own commit.